vizro-core/docs/pages/user-guides/data.md

```diff
@@ -188,11 +188,11 @@ that accepts different SQL queries as arguments.
     [DataConnector]: ../../assets/user_guides/data/data_selected_from_source.png
 
 
-### Kedro data catalog
+### Kedro integration
 
 If the data you are visualizing is a [`kedro_datasets.pandas`](https://docs.kedro.
-org/en/stable/kedro_datasets.html) type from a Kedro data catalog, you can leverage
-Vizro's [Kedro integration](integration.md#kedro) to connect your charts to the data catalog.
+org/en/stable/kedro_datasets.html) type from a Kedro data catalog, you can use
+Vizro's [Kedro integration](integration.md#kedro) to connect your charts to [Kedro's Data Catalog](https://docs.kedro.org/en/stable/data/index.html).
```

Comment:
```suggestion
Vizro's [Kedro integration](integration.md#kedro) to connect your charts to [Kedro's data catalog](https://docs.kedro.org/en/stable/data/index.html).
```

Reply:
Sadly this is capitalized according to Kedro style, so I'll leave it as I corrected.